lib/getSrcForJoin.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * Retrun the join src jobs given a workflowGraph and dest job
+ * @method getSrcForJoin
+ * @param  {Object}    workflowGraph    Directed graph representation of workflow
+ * @param  {Object}    config
+ * @param  {String}    config.jobName   The dest job name to be triggered after a join
+ * @return {Array}                      List of node object consists of job name and id
+ */
+const getSrcForJoin = (workflowGraph, config) => {
```

Comment:
why not just have `jobName` as a `string` instead of `config` here? The only field we need is `jobName`

Reply:
=.= ok